strategy.ini

```diff
@@ -0,0 +1,90 @@
+; some useful links:
+; - https://janelia-flyem.github.io/licenses.html
+; - https://dwheeler.com/essays/floss-license-slide.html
+
+; Authorized and unauthorized licenses in LOWER CASE
+[Licenses]
+authorized_licenses:
+	; aliases for MIT License
+	MIT
+	MIT license
+	https://opensource.org/licenses/MIT
+	License :: OSI Approved :: MIT
+
+	; aliases for BSD License (and variants)
+	BSD
+	BSD license
+	new BSD
+	(new) BSD
+	new BDS license
+	simplified BSD
+	3-Clause BSD
+	BSD-3-Clause
+	BSD 3-Clause
+	BSD-2-Clause
+	BSD-like
+
+	; aliases for Apache License version 2.0
+	Apache 2.0
+	Apache License 2.0
+	Apache License, Version 2.0
+	Apache License Version 2.0
+	ASL 2
+;	some packages use 'Apache Software' as license string,
+;	which is ambiguous. However, 'Apache Software'
+;	will likely match with 'Apache 2.0'
+	Apache Software
+	BSD, Public Domain, Apache
+	http://www.apache.org/licenses/LICENSE-2.0
+
+;	PSF (BSD-style)
+	Python Software Foundation
+	PSF
+
+	; other permissive licenses
+	Historical Permission Notice and Disclaimer (HPND)
+	HPND
+	ISC
+	BSD or Apache License, Version 2.0
+	Modified BSD
+	Expat
+
+unauthorized_licenses:
+;	aliases for MPL 2.0
+	MPL-2.0
+	MPL 2.0
+	Mozilla Public License 2.0 (MPL 2.0)
+
+;    Section 8 of https://www.mozilla.org/en-US/MPL/2.0/Revision-FAQ/
+	MPL 1.1
+	MPL-1.1
+
+;	http://www.gnu.org/licenses/license-list.en.html#apache2
+	GPLv2
+	GPLv2+
+	GNU General Public License v2 or later (GPLv2+)
+
+;	LGPL
+	LGPL
+	GNU Library or Lesser General Public License (LGPL)
+
+;	LGPLv2.1
+	LGPLv2.1
+	LGPLv2.1+
+
+;	LGPLv3
+	GNU Lesser General Public License v3 (LGPLv3)
+	LGPLv3
+
+;	GPL v3
+	GPL v3
+	GPLv3+
+
+[Authorized Packages]
+gym: >=0.15
+;filelock is public domain
+filelock: >=3.0.12
+fetchai-ledger-api: >=0.0.1
+chardet: >=3.0.4
+certifi: >=2019.11.28
```

Comment:
Although both `certifi` and `chardet` are licensed with a more restrictive license, they are not a first level dependency. Instead, they both are used by requests.